rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html

```diff
@@ -0,0 +1,41 @@
+            </p>
+            <p>
+                When a student requests to join your class, a request notification will be sent to your administration panel. To go to your administration panel, go to the classrooms page. Your own classrooms will be at the top of the list. Click "Admin" to the right of your classroom title. Under the member-request panel, all people who have asked to join your classroom await approval. To group approve, click their checkmark above their icon for as many as your want to approve, then under the "action" dropdown choose "Approve" and click "Submit all changes". They will promoted to the "students" panel. As a student, when they save their project, there will be a bar where they can choose whether to save to their classroom. They will always want to use this option, which requires them to highlight the name of their classroom. Teach them how to use this feature to prevent projects being saved to the public pool instead of your classroom pool.
+            </p>
+            <p>
+                After a student saves a project and it is still unapproved, they are free to modify it as much as they like. When they are done working on it, they will have to go to their "My Profile" page and scroll down to the bottom where "Projects pending approval" are. There they click on the project, where a setting panel pops up. Here they can add a description, ensure that it is saving to your classroom, and so forth. Then they click "publish" at the very bottom. 
+            </p>
+            <p>
+                After a student publishes a project to their classroom, it will show up under the "Approval-requests" panel in the classroom administrator. It is not visible to the students yet. Here you can see a screenshot of the project, to deem whether it is classroom appropriate or not. If it is, you can check it and then under the "action" dropbox choose "approve". This will promote it to the "approved-objects" pool, where it will be visible to other students in the classroom. It will display their name underneath the project as well as a description, if the student inputs one. This allows you to curate the presentation of projects, so that what is seen by your students is verified appropriate. We at CSDT do not approve projects, so it will be your responsibility to check in to approve new projects if you do not want them to languish in the approval queue. 
```

Comment:
Awkward: "in the classroom administrator" -> "of the classroom administration page"
Don't go by just the photo: "Here you can see a screenshot of the project" -> "Here you can see a screenshot, or click to view the project as a whole."